### ISSUE TITLE
Selecting a filter does not constitute a corresponding adjustment/addition to a posting data record after deleting the frontend cache

### ISSUE THREAD

**Issue by zenoussi** [itx-informationssysteme/jobapplications#134]
1. filling in the [division] field in the posting data record
2. flush frontend caches of the page for the list display or all pages
3. the addition/change is not taken into account in the "Division" filter of the list display
4. flush all caches
5. the addition/change is now displayed

- jobapplications 3.0.4
- TYPO3 12.4.8
- PHP 8.1.13

Greetings
Ulli

**Comment by itx-benjamin-jasper** [comment]
Yes the behaviour is considered normal, because the filter options are being cached, so they don't have to be cached for every filter request.

We could think about a hook that automatically clears the cache based on some cache tag, but this is not a priority for us right now.
